code_samples/back_office/image_editor/config/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const Encore = require('@symfony/webpack-encore');
+const path = require('path');
+const getEzConfig = require('./ez.webpack.config.js');
+const eZConfigManager = require('./ez.webpack.config.manager.js');
+const eZConfig = getEzConfig(Encore);
+const customConfigs = require('./ez.webpack.custom.configs.js');
+
+Encore.reset();
+Encore
+    .setOutputPath('public/build/')
+    .setPublicPath('/build')
+    .enableStimulusBridge('./assets/controllers.json')
+    .enableSassLoader()
+    .enableReactPreset()
+    .enableSingleRuntimeChunk()
+    .copyFiles({
+        from: './assets/images',
+        to: 'images/[path][name].[ext]',
+        pattern: /\.(png|svg)$/
+    })
+    .configureBabel((config) => {
+        config.plugins.push('@babel/plugin-proposal-class-properties');
+    })
+
+    // enables @babel/preset-env polyfills
+    .configureBabelPresetEnv((config) => {
+        config.useBuiltIns = 'usage';
+        config.corejs = 3;
+    })
+;
+
+// Welcome page stylesheets
+Encore.addEntry('welcome_page', [
+    path.resolve(__dirname, './assets/scss/welcome-page.scss'),
+]);
+
+Encore.addEntry('app', './assets/app.js');
+
+const projectConfig = Encore.getWebpackConfig();
+module.exports = [ eZConfig, ...customConfigs, projectConfig ];
+
+eZConfigManager.add({ 
```

Comment:
This should be somewhere before the `module.exports = [ eZConfig, ...customConfigs, projectConfig ];`